app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+  enum :role, %i[freelancer client]
+  validates :role, presence: true
+  validates :user_name, presence: true
```

Comment:
Move this presence validation in one line.

Reply:
ok